src/main/kotlin/com/terraformation/backend/tracking/db/T0Store.kt

```diff
@@ -540,22 +457,168 @@ class T0Store(
     }
   }
 
-  private fun plotHasCompletedObservations(
-      monitoringPlotIdField: Field<MonitoringPlotId?>,
-      isPermanent: Field<Boolean?>,
-  ): Condition =
-      DSL.exists(
-          DSL.selectOne()
-              .from(OBSERVATION_PLOTS)
-              .where(
-                  OBSERVATION_PLOTS.MONITORING_PLOT_ID.eq(monitoringPlotIdField)
-                      .and(OBSERVATION_PLOTS.IS_PERMANENT.eq(isPermanent))
-                      .and(OBSERVATION_PLOTS.COMPLETED_TIME.isNotNull)
+  private fun getAllPermPlotSpeciesT0Data(plantingSiteId: PlantingSiteId): Pair<Boolean, Boolean> {
```

Comment:
I didn't expect a function with this name to return a pair of booleans, but rather a list of all the species T0 data from permanent plots. A comment explaining the return value would help.

Reply:
Wrote that comment before seeing the next one, which simplifies the retrieval of `survivalRateIncludesTempPlots` and makes this one unnecessary